app/posts/find-teacher-training/2021-08-16-standardised-entry-requirement-descriptions.md

```diff
@@ -0,0 +1,61 @@
+
+> GCSE in English (or approved equivalent) at grade C/Level 4 or above GCSE in Mathematics (or approved equivalent) at grade C/Level 4 or above GCSE in Science (or approved equivalent) at grade C/Level 4 or above
+
+These minor inconsistencies may be confusing for candidates, and the requirements weren’t always written in the easiest to understand way.
+
+Instead, for the 2022 to 2023 cycle, we have changed the Publish interface to only ask providers to specify the entry requirements that may differ across courses, namely the [acceptance of pending GCSEs and GCSE equivalency tests](/publish-teacher-training-courses/pending-gcses-equivalency-tests/) and the [minimum degree class required](/publish-teacher-training-courses/degree-entry-requirements/).
+
+These changes allow us use consistent language to describe the standard entry requirements.
+
+## New content structure
+
+### Primary courses
+
+The standardised content is:
+
+> An undergraduate degree, or equivalent.
+>
+> GCSE grade 4 (&#8203;C) or above in English, maths and science, or equivalent qualification.
+
+Then the following paragraph is constructed from the provider’s answers to the pending GCSEs and equivalency tests questions, for example:
+
+> We’ll consider candidates who are currently taking GCSEs. We will not consider candidates who need to take GCSE equivalency tests.
+
+or
+
+> We’ll consider candidates who are currently taking GCSEs. We’ll consider candidates who need to take a GCSE equivalency test in English, maths, or science.
+
+Where providers do accept equivalency tests, they are asked to give details, and this free text field is shown at the end.
+
+### Secondary courses
+
+For secondary courses, a standardised paragraph is added about the degree subject:
+
+> Your degree subject should be in [course subject] or a similar subject. Otherwise you’ll need to prove your subject knowledge in some other way.
+
+Providers are able to add additional text after this if they have further degree subject requirements, such as a percentage of modules being in the relevant subject.
+
+The GCSE requirements for Secondary courses do not include science:
+
+> GCSE grade 4 (&#8203;C) or above in English and maths, or equivalent qualification.
+
+## Findings from user research
+
+There was consistent confusion around the term 'equivalent', as it is used both in 'equivalency test' and 'or equivalent', but these mean different things. To improve this, we have qualified these terms as 'GCSE equivalency test' and 'or equivalent qualification'.
+
+From speaking to providers, we learnt that there is a wide variation in how GCSE equivalency tests are handled, with some running these in-house and others requiring specific third-party companies. This meant that we could not use terms like "offered" or "accepted". Instead the language refers to considering candidates who need to take the a GCSE equivalency test, with the free text paragraph after giving details.
```

Comment:
Suggestion: break up the first sentence, as below.

'We learnt that providers handle GCSE equivalency tests differently. Some providers run them in-house, others only accept tests from specific third-party companies.'